src/types/actions/interactive-message.ts

```diff
@@ -59,6 +59,13 @@ export interface InteractiveMessage<Action extends InteractiveAction = Interacti
   message_ts?: string;
   // NOTE: the original_message is not available from ephemeral messages
   original_message?: { [key: string]: string; };
+
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
Haven't confirmed if the payload is actually getting these two keys yet in enterprise install (and that team is null)